interfaces/HO-Portal/src/app/program/program-payout/program-payout.component.ts

```diff
@@ -17,15 +17,14 @@ import { ProgramPhase } from 'src/app/models/program.model';
 export class ProgramPayoutComponent implements OnChanges {
   @Input()
   public programId: number;
-
   @Input()
   public selectedPhase: string;
-
   @Input()
   public transferValue: any;
-
   @Input()
   public currencyCode: string;
+  @Output()
+  emitCompleted: EventEmitter<boolean> = new EventEmitter<boolean>();
```

Comment:
I would suggest to name these emitters not with an `emit` verb.
Something like `isCompleted` makes more sense when you use something to 'listen' to this event.